### PR TITLE
[DOC] Changes to first text

### PR DESCRIPTION
**Pull request by TonyBagnall**
this tidies up shapelet notebooks, but main change is on the text

image

this wording does not scan for me. I also prefer toolkit to framework. Changed to

`aeon` is a scikit-learn compatible toolkit for time series machine learning tasks
such as anomaly detection, classification, clustering, forecasting, regression,
segmentation and similarity search.</p>

- We provide a broad library of time series algorithms, including the latest
  state of the art for many tasks.
- Our algorithms are all implemented as efficiently as possible by, for example, using `numba`.
- `aeon` is built on top of `scikit-learn`, allowing for easy integration with other
  machine learning libraries and other time series packages.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -4,15 +4,16 @@ hide-toc: true
...
-- Interfaces with other time series packages to provide a single framework for algorithm
-comparison.
-- Built on top of `scikit-learn`, allowing for easy integration with other machine
-learning libraries.
+`aeon` is a scikit-learn compatible toolkit for time series machine learning tasks
+such as anomaly detection, classification, clustering, forecasting, regression,
+segmentation and similarity search.</p>
+
+- We provide a broad library of time series algorithms, including the latest
+  state of the art for many tasks.
```
Comment (MatthewMiddlehurst) [inline docs/index.md:12]: "We provide a broad library of time series algorithms, including the latest advances and state-of-the-art for many tasks." maybe?

File: docs/index.md [hunk 2]
```diff
@@ -4,15 +4,16 @@ hide-toc: true
...
+- Our algorithms are all implemented as efficiently as possible by, for example,
+  using `numba`.
```
Comment (MatthewMiddlehurst) [inline docs/index.md:14]: Wouldn't say all

File: docs/index.md [hunk 3]
```diff
@@ -4,15 +4,16 @@ hide-toc: true
 
 # Welcome to __aeon__
 
-`aeon` is a scikit-learn compatible toolkit for time series tasks such as
-classification, clustering, segmentation and anomaly detection.</p>
-
-- Provides a broad library of time series algorithms, including the latest advances.
-- Efficient implementation of time series algorithms using `numba`.
-- Interfaces with other time series packages to provide a single framework for algorithm
-comparison.
```
Comment (MatthewMiddlehurst) [inline docs/index.md:13]: Can understand the apprehension with "framework", but this is something we do in most modules i.e. MrSQM, TSFresh, multiple anomaly detectors etc.

File: examples/classification/SastVsViz.ipynb
Comment (MatthewMiddlehurst) [inline examples/classification/SastVsViz.ipynb:1]: not looked at the notebook changes yet. Was some of this content moved to shapelets or just removed?

Comment (TonyBagnall) [reply examples/classification/SastVsViz.ipynb:1]: there was a loose notebook moved to shapelets. Its a bit repetitive, but thought keep the content

File: docs/index.md (hunk 3 above)
Comment (TonyBagnall) [reply docs/index.md:13]: is this not covered by
"- `aeon` is built on top of `scikit-learn`, allowing for easy integration with other
  machine learning libraries and other time series packages."

Comment (TonyBagnall) [reply docs/index.md:13]: added this
We provide a range of tools for reproducing benchmarking results and evaluating time
  series algorithms implemented in `aeon` and other `scikit-learn` compatible packages.

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**